src/styles/header.scss

```
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    height: 40px;
+    left: 50px;
+
+    background-image: url(../images/logo.png);
+  }
+}
```

Use padding prop to set margins around the edges
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 60px;
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
  padding: 0 50px;

  &__logo {
    width: 40px;
    height: 40px;
    
    background-image: url(../images/logo.png);
  }
}
```